### ISSUE TITLE
error when trying to run locally on osx

### ISSUE THREAD

**Issue by vesper8** [agoldis/sorry-cypress#1]
I'm trying to get this to work on my Macbook. I followed instructions, and get this error when I run `yarn start`

```
yarn run v1.17.3
$ node --experimental-modules .
(node:92497) ExperimentalWarning: The ESM module loader is experimental.
internal/modules/esm/default_resolve.js:59
  let url = moduleWrapResolve(specifier, parentURL);
            ^

Error: Cannot find module '/DEV/_CYPRESS/sorry-cypress' imported from /DEV/_CYPRESS/sorry-cypress/
    at Loader.resolve [as _resolve] (internal/modules/esm/default_resolve.js:59:13)
    at Loader.resolve (internal/modules/esm/loader.js:73:33)
    at Loader.getModuleJob (internal/modules/esm/loader.js:149:40)
    at Loader.import (internal/modules/esm/loader.js:133:28)
    at internal/modules/cjs/loader.js:830:27
    at processTicksAndRejections (internal/process/task_queues.js:85:5) {
  code: 'ERR_MODULE_NOT_FOUND'
}
error Command failed with exit code 1.
```

Any idea what might be wrong?

**Comment by agoldis** [comment]
I've updated the repo and removed the use of experimental modules - it's all old commonjs now. Please pull and try again

**Comment by vesper8** [comment]
thank you for your quick fix, it works now!

However.. I think I must have it wrong still

I only have one machine on which to perform tests right now. If I run `cypress run --parallel --record --key xx --ci-build-id local-000 --spec="tests/e2e/specs/**/*"`

it finds 4 specs and runs them supposedly in parallel, however it seems to take just as much time as if I run it without the parallel flag. Am I misunderstanding? Can this speed up running multiple specs on a single dev machine or it only works in the context of CI and using multiple remote CI servers?

**Comment by agoldis** [comment]
TL;DR

You'd run two or more `cypress run` instances to see parallelization.

LR:
Each instance of command `cypress run` represents a "machine" - the parallelisation is applied on "machine" (or process) level. 

To see it in action, run 2 commands `cypress run` in different terminals - you will see them talking to the service and getting different tasks - that's how paralellisation is achieved. 

When you run just a single instance of `cypress run` - all the specs are run one by one by this single instance.

For comparison, imagine a CI system that launches 100 "machines" to run the tests - each machine is not aware of another, but they share the same `--ci-build-id` and talk to the service (either the original cypress dashboard or an alternative). The service receives the requests from those 100 machine and assignes each what task to run. 
After completion of a task each machine sends a request to a service to get the next task.... until all no task returned by the service - i.e. all tasks completed.

Hope that helps :)

**Comment by vesper8** [comment]
Thanks for that! It helps! But it also brings up a lot of new questions... not to mention that it looks like it doesn't work on my machine.

When I try to run the same command as seen in my post above in two different terminals, one terminal ends up doing 1 or 2 and the other terminal prints out this and immediately exits

```
  (Run Starting)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Cypress:    3.4.1                                                                              │
  │ Browser:    Electron 61 (headless)                                                             │
  │ Specs:      4 found (forms/team/failure/adds-team.js, forms/team/failure/modifies-team.js, fo… │
  │ Searched:   tests/e2e/specs/forms/team/**/*                                                    │
  │ Params:     Group: false, Parallel: true                                                       │
  │ Run URL:    https://sorry.cypress.io/                                                          │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘


====================================================================================================

  (Run Finished)
```

Another question is.. could this unofficial parallelization api be applicable through Jenkins? Like, I'd need to modify the cypress config on each of my remote build servers to have them all point at one api endpoint and then as they would run the battery of cypress tests on each server, they'd end up sharing the load and intelligently spreading specs across all servers? Easy-peasy? =D
